Assets/Scripts/ThrowController.cs

```diff
@@ -65,10 +65,13 @@ public Vector3 GetMousePosition()
                 (Utils.InputAxisInUse(Constants.INPUT_AXIS_HORIZONTAL_RIGHT_STICK) || 
                  Utils.InputAxisInUse(Constants.INPUT_AXIS_VERTICAL_RIGHT_STICK)))
             {
-                float horizontal = 10 * sensitivityController * Input.GetAxis(Constants.INPUT_AXIS_VERTICAL_RIGHT_STICK);
-                float vertical = 10 * sensitivityController * Input.GetAxis(Constants.INPUT_AXIS_HORIZONTAL_RIGHT_STICK);
+                float horizontal = 10 * sensitivityController * Input.GetAxis(Constants.INPUT_AXIS_HORIZONTAL_RIGHT_STICK);
+                float vertical = 10 * sensitivityController * Input.GetAxis(Constants.INPUT_AXIS_VERTICAL_RIGHT_STICK);
+
+                Quaternion rotatorToCamera = Quaternion.Euler(0, mainCamera.transform.eulerAngles.y, 0);
+                Vector3 rotatedDirection = rotatorToCamera * new Vector3(horizontal, 0, vertical);
                 
-                return new Vector3(position.x + horizontal, position.y, position.z + vertical);
+                return new Vector3(position.x + rotatedDirection.x, position.y, position.z + rotatedDirection.z);
```

Comment:
I wonder if it would be possible to get the ThrowController and MovementController to use the same logic for figuring out this camera-relative axis? MovementController is keeping track of it already so maybe that can be exposed in a public getter or something.

Reply:
Yeah I'll keep track of the camera orientation in PlayerController so both classes can access it from there!